packages/errors/README.md

```diff
@@ -45,6 +46,32 @@ export function multipleImports() {
 }
 ```
 
+### Test Utilities
+
+You can use the `expectToThrowFuelError` test utility when asserting errors.
+
+```ts
+// can also be imported from `fuels` when using the umbrella package
+import { expectToThrowFuelError } from "@fuel-ts/errors/test-utils";
+
+import { myFn } from "...";
+
+describe('this and that'm () => {
```

Comment:
```suggestion
describe('this and that' () => {
```

Reply:
Another case for test backed snippets 🤔  Potentially overcomplicating a simple issue but we use a markdown generator to pass tests to files.

Reply:
That's a good point, although we'd need to develop a routine to validate the READMEs.

I'm resolving this for now as it stands outside the scope of this PR.